Symfony/CS/Fixer/PSR2/ElseifFixer.php

```diff
@@ -22,12 +22,28 @@
 class ElseifFixer extends AbstractFixer
 {
     /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        # handle `T_ELSE T_WHITESPACE T_IF` treated as single `T_ELSEIF` by HHVM
+        # see https://github.com/facebook/hhvm/issues/4796
+        if ($tokens->isTokenKindFound(T_ELSEIF)) {
```

Comment:
you should apply this logic only for HHVM IMO, given that the fixing logic is also done only for HHVM